### ISSUE TITLE
Request that the windows installer has a choice to put VST 2 in a custom folder

### ISSUE THREAD

**Issue by atomoutput** [DISTRHO/Cardinal#382]
.

**Comment by falkTX** [comment]
Can you give more details? why is this needed?

**Comment by atomoutput** [comment]
far from being a deal-breaker, it's just a standard practice all vst installers have, so why not?

**Comment by falkTX** [comment]
I dont use Windows though, so I am not used to this practice.
https://helpcenter.steinberg.de/hc/en-us/articles/115000177084-VST-plug-in-locations-on-Windows seems to indicate that indeed, this is not well defined... 🤦 

But if you need to install cardinal to specific locations, why not then use the zip download/release?

**Comment by atomoutput** [comment]
i am, it really isn't a problem for me but im guessing less computer-savvy people will be confused about how to do it, mostly because i think this is the only installer i've seen that doesn't have that.

**Comment by falkTX** [comment]
right. to be honest, this is something I will leave out for someone else to do.
the installer files are in https://github.com/DISTRHO/Cardinal/blob/main/utils/inno/win64.iss so anyone can do a patch / pull request with the changes needed for this.

Initially I didn't even want to do the installers, but quite a few people complained so I added them in 22.05 or sometime around that.
I do not need them or use them at all (only do tests to ensure files are installed and updated correctly).

So basically, contributions welcome :)

**Comment by falkTX** [comment]
This has been added, will be present for 23.04 release

![image](https://user-images.githubusercontent.com/1334853/233616067-b57551f1-23d2-4a15-aae2-cc7f8c9d9852.png)
